JitsiConference.js

```diff
@@ -2508,13 +2499,6 @@ JitsiConference.prototype.isStartVideoMuted = function() {
     return this.startVideoMuted;
 };
 
-/**
- * Returns measured connectionTimes.
- */
-JitsiConference.prototype.getConnectionTimes = function() {
-    return this.room.connectionTimes;
```

Comment:
I see this method is used in jitsi-meet. Aren't we going to brake something? 

I think it will be good to expose those measurements somehow. WDYT? Maybe they can be used for performance testing, etc.

Reply:
I plan to fix JM too. The `perfMetrics` object is exposed in JitsiMeetJS, but the metrics can be obtained with performance.getEntries(). I can create a helper for that.